wegas-app/src/main/webapp/2/src/API/websocket.ts

```diff
@@ -121,14 +158,55 @@ export default class WebSocketListener {
               //pusher events
               return;
             }
-            this.eventReveived(processed.event, processed.data);
+            this.eventReveived(
+              processed.event as WebSocketEvent,
+              processed.data,
+            );
           },
         ),
       );
     });
   }
-  private eventReveived(event: string, data: any) {
-    console.log(event, data);
+  public bindCallback(
+    eventId: WebSocketEvent,
+    callback: (
+      data: any, //eslint-disable-line @typescript-eslint/no-explicit-any
+    ) => void,
+  ) {
+    if (this.events[eventId]) {
+      this.events[eventId].push(callback);
+    } else {
+      console.log('Unknown event'); //eslint-disable-line no-console
```

Comment:
Let eslint warnings while we develop. It may be useful to find them afterwards
```suggestion
      console.log('Unknown event');
```

Reply:
done